service/src/main/java/bio/terra/workspace/app/configuration/external/WsmApplicationConfiguration.java

```diff
@@ -19,20 +17,11 @@
 @ConfigurationProperties(prefix = "workspace.application")
 public class WsmApplicationConfiguration {
   public static class App {
-    private String identifier;
     private String name;
```

Comment:
Do these need to match the config file's order? Document if so. Also, how does `state` get populated for the test app?

Reply:
They do not need to match. Spring is doing the mapping work here.

The application.yml file has an optional include for a write-config.sh output file. That lets us inject the test application and the Azure config in the local configuration. So when we run `local` configuration tests the application is there.